crt_portal/cts_forms/admin.py

```diff
@@ -211,12 +212,19 @@ def voting_toggle_display_name(self, obj):
 
 
 class CampaignAdmin(admin.ModelAdmin):
+    class Media:
+        js = ('js/admin_copy.js',)
+        css = {
+            'all': ('css/compiled/admin.css',)
+        }
+
     list_display = ['uuid', 'internal_name', 'campaign_url']
     readonly_fields = ['campaign_url']
 
     @admin.display(description='Campaign URL')
     def campaign_url(self, obj):
-        return obj.get_absolute_url()
+        url = obj.get_absolute_url()
+        return mark_safe(f'<input disabled="disabled" class="admin-copy absolute-url" value="{url}"/>')
```

Comment:
When running Waves on the admin page to check for a11y issues, it flagged 1 error, and that was that this <input> element did not have an associated label. I'm not sure if disabling the input affects the need for a label, but I would slap one on there for good measure.